infrastructure/repository/authzed/LocalSpiceDbContainer.go

```diff
@@ -0,0 +1,105 @@
+//go:build !release
```

Comment:
this together with the appended build tag in make and pipelines makes sure the test dependencies don't get compiled into the binary. Found out on the way that we also have (unnecessary)  testify dependencies introduced by spicedb, see thread here: https://discord.com/channels/844600078504951838/1101449257623502959